pyclam/tests/test_functional.py

```diff
@@ -2,35 +2,38 @@
 
 import numpy as np
 
-from pyclam import criterion, Manifold
-from pyclam.tests.utils import linear_search
+from pyclam import Manifold
 
 
+# TODO: Should we scrap this file?
```

Comment:
Should we?

Reply:
yes and no. We should have functional tests but applying them to manifold makes less and less sense as we cut that down to a lower-level library. Our functional tests should probably be written around chaoda and search

Reply:
Functional tests should verify that users can accomplish larger goals, rather than unit tests which really just focus on one incremental piece.

Reply:
Agreed. We can handle this in a later PR